data/tutorials/gs_00_up_and_running.md

```diff
@@ -41,7 +52,7 @@ methods.](https://opam.ocaml.org/doc/Install.html)
 **Install Compiler**
 It's essential to install the OCaml compiler because OCaml is a compiled language, so please don't skip this step. The first part sets up Opam, and the second part installs the OCaml base compiler:
```

Comment:
```suggestion
It's essential to install the OCaml compiler because OCaml is a compiled language, so please don't skip this step. The first part sets up opam, and the second part installs the OCaml base compiler:
```